plugin/evm/sync/syncervm_client.go

```diff
@@ -17,69 +18,91 @@ import (
 	"github.com/ava-labs/coreth/core/state/snapshot"
 	"github.com/ava-labs/coreth/eth"
 	"github.com/ava-labs/coreth/params"
-	"github.com/ava-labs/coreth/plugin/evm/atomic/state"
 	"github.com/ava-labs/coreth/plugin/evm/message"
-	syncclient "github.com/ava-labs/coreth/sync/client"
 	"github.com/ava-labs/coreth/sync/statesync"
 	"github.com/ava-labs/libevm/common"
 	"github.com/ava-labs/libevm/core/rawdb"
+	"github.com/ava-labs/libevm/core/types"
 	"github.com/ava-labs/libevm/ethdb"
 	"github.com/ava-labs/libevm/log"
 )
 
-const (
-	// State sync fetches [parentsToGet] parents of the block it syncs to.
-	// The last 256 block hashes are necessary to support the BLOCKHASH opcode.
-	parentsToGet = 256
-)
+// ParentsToFetch is the number of the block parents the state syncs to.
+// The last 256 block hashes are necessary to support the BLOCKHASH opcode.
+const ParentsToFetch = 256
 
 var stateSyncSummaryKey = []byte("stateSyncSummary")
 
-// stateSyncClientConfig defines the options and dependencies needed to construct a StateSyncerClient
-type stateSyncClientConfig struct {
-	enabled    bool
-	skipResume bool
+type BlockAcceptor interface {
+	PutLastAcceptedID(ids.ID) error
+}
+
+// EthBlockWrapper can be implemented by a concrete block wrapper type to
+// return *types.Block, which is needed to update chain pointers at the
+// end of the sync operation.
+type EthBlockWrapper interface {
+	GetEthBlock() *types.Block
+}
```

Comment:
This type is duplicated between here and `plugin/evm/block.go` - do we need both of these definitions?

Reply:
removed the one in block